lib/stores/accounts_store.dart

```diff
@@ -91,6 +88,8 @@ abstract class _AccountsStore with Store {
   void load() async {
     final prefs = await SharedPreferences.getInstance();
 
+    // I barely understand what I did. Long story short it asserts a 
```

Comment:
did you mean _inserts_ instead of _asserts_?

Reply:
I meant `casts`